server/test/handlers/account.test.js

```diff
@@ -4,39 +4,112 @@ describe('Test to check valid account creation', () => {
     beforeEach(() => {
         jest.resetModules();
     });
-    
+    //TEST 1
     //Create a test using Arrange, Act, Assert format
-    test('Initial set up: Checking if the account creation details are correct', async () => {
+    test('rejection of null values for student_id, first_name, last_name, email, type, phone, school_id', async () => {
         // Arrange
-
-        // In order to test, the tester should input these credentials
-        const email = 'victorhugo@gmail.com';
-        const password = 'victorlovespoetry';
-        const firstName = 'Victor';
-        const lastName = 'Hugo';
-        const phone = '6476442200';
-        const type = 'student';
-        const studentID = '20764242';
-        
-
-        const testUser = {
-            type: 'student', //note: 'type' is called 'role' in the UI, but refers to either a worker or a student.
-            studentId: '20764242',
-            firstname: 'Victor',
-            lastname: 'Hugo',
-            email: 'victorhugo@gmail.com',
-            phone: '6476442200'
-        };
+        const testStudent_id = null;
```

Comment:
Hi Nancy,
I would recommend renaming these variables to be lowerCamelCase, e.g., `testStudentId` rather than using underscores to increase code consistency.

Reply:
Hi Amy,
Thank you for catching that!
My action: I've renamed all relevant variables in `server/test/handlers/account.test.js` to be in camelCase.

You can now see these changes in:
lines 30-36 of server/routes/public.js
lines 71-78 of server/routes/public.js
lines 92-100 of server/routes/public.js

Please confirm if this takes care of your suggestion!